webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './src/index',
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        loader: 'babel',
+        exclude: /node_modules/,
+        query: {
+          presets: ['es2015', 'stage-1']
+        }
+      }
+    ]
+  },
+  output: {
+    filename: 'dist/denormalizr.min.js',
+    libraryTarget: 'umd',
+    library: 'denormalizr'
+  },
+  plugins: [
+    new webpack.optimize.OccurenceOrderPlugin(),
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': JSON.stringify('production')
+      }
+    }),
```

Comment:
We want to have precompiled production and development UMD builds in the `dist` folder. You are probably right with `OccurenceOrderPlugin`, I was just copying webpack.config from different very simple setup. Do you want me to remove it ?